### PR TITLE
Traceable PeerConnection improvements.

### PR DESCRIPTION
**Pull request by gpolitis**
- Adds a transformation history that keeps track of the SDP modifications.
- Makes it possible to configure the number of simulcast layers through config.js

### CODE REVIEW

File: modules/xmpp/TraceablePeerConnection.js [hunk 1]
```diff
@@ -593,26 +658,62 @@ TraceablePeerConnection.prototype.createAnswer
...
 
             if (!RTCBrowserType.isFirefox())
             {
                 answer = self.ssrcReplacement(answer);
-                self.trace('createAnswerOnSuccess::mungeLocalVideoSSRC',
-                    dumpSDP(answer));
+
+                if (self.history) {
+                    // Add a transformation in the point.
+                    point.transformations.push({
```
Comment (hristoterezov) [inline modules/xmpp/TraceablePeerConnection.js]: Shouldn't you define "point" here?

Comment (bgrozev) [inline modules/xmpp/TraceablePeerConnection.js]: Or rather define it once -- all these "var point"s are in the same scope.

Comment (gpolitis) [inline modules/xmpp/TraceablePeerConnection.js]: Thanks @hristoterezov, @bgrozev ! I'll fix this with a few more changes that I'm preparing.

Comment (lyubomir) [inline modules/xmpp/TraceablePeerConnection.js]: Yup, as @bgrozev said, the multiple var declarations are automatically hoisted by JS into one to the top so it doesn't matter (to JS) where the 'var point's  are and a single one anywhere in the function is enough.

**Pull request closed without merging**